app/util/conf/base_config_loader.py

```diff
@@ -162,6 +166,11 @@ def _get_config_file_whitelisted_keys(self):
             'default_http_timeout',
             'https_cert_file',
             'https_key_file',
+            'pagination_offset',
+            'pagination_limit',
+            'pagination_max_limit',
```

Comment:
Oh I was under the impression we'd want to be able to customize them in the config. I can remove them from this list